__mocks__/fixtures/confirmTeamInvitation.ts

```diff
@@ -0,0 +1,128 @@
+                      cover: {
+                        text: "#red",
+                        accent: "#red",
+                        primary: "#red",
+                        background: "#red",
+                      }
+                    },
+                    slug: 'example-community',
+                    active: true,
+                    description: 'A detailed description of the community.',
+                    metadata: {
+                        invite_id: 'invite123',
+                        submissions: 5,
+                        bestSubmissions: ['submission1', 'submission2'],
+                        feedbacks: 10,
+                        name: 'Example Certificate',
+                        issuedOn: '2024-01-01',
+                        image: 'http://example.com/certificate-image.jpg',
+                        title: 'Certificate of Achievement',
+                        description: 'A detailed description of the certificate.',
+                        narrative: 'Narrative about the achievement.',
+                        recipientName: 'John Doe',
+                        issuerName: 'Example Organization',
+                        comment: 'Congratulations on your achievement!',
+                        linkToWork: 'http://example.com/work',
+                        team: {
+                          members: ['John Doe', 'Jane Doe'],
+                          name: 'Team Example',
+                        },
+                    reward: {
+                      id: 'reward123',
+                      name: 'Special Reward',
+                      description: 'A special reward for community members.',
+                    },
+                    courses: 5,
+                    challenges: 10,
+                    duration: 30, // Duration in days
+                    items: ['item1', 'item2'], // Assuming items can be represented as strings
+                    challenge: {
+                      id: 'challenge1',
+                      title: 'First Challenge',
+                      description: 'Description of the first challenge.',
+                    },
+                    submission: {
+                      id: 'submission1',
+                      title: 'First Submission',
+                      content: 'Content of the first submission.',
+                    },
+                    can_mint_certificates: true,
+                  },    
+                score: 75,
+              },
+            },
+            discordConnected: true,
+            isKycVerified: false,
+            feedbacks: {
+              score: 5,
+              comment: 'Great service!',
+            },
+            discord: {
+              connected: true,
+            },
+            kycStatus: KYCSTATUS.PENDING,
+            referrals: [
+              {
+                referrerId: 'ref123',
+                referredId: 'ref456',
+                reward: 50,
+              },
+            ],
+          },
```

Comment:
We arleady have invite mock defined here: __ mocks__/fixtures/challenge.ts : line 119